test/lit/wat-kitchen-sink.wast

```diff
@@ -103,13 +105,31 @@
  ;; CHECK:      (global $i32 i32 (i32.const 42))
  (global $i32 i32 i32.const 42)
 
+ ;; memories
+ ;; CHECK:      (memory $mem 1 1)
+ (memory $mem 1 1)
+ (memory 0)
+ ;; CHECK:      (memory $0 0)
+
+ ;; CHECK:      (memory $mem-i32 0 1)
+ (memory $mem-i32 i32 0 1)
+ ;; CHECK:      (memory $mem-i64 i64 2)
+ (memory $mem-i64 i64 2)
+ (memory (export "mem") (export "mem2") (import "" "mem") 0)
+ ;; CHECK:      (memory $mem-init 1 1)
+ (memory $mem-init (data "hello, world!"))
```

Comment:
The data segment not being added in the output is what the description mentioned as a TODO, I guess?

Reply:
Yeah, exactly. There's a TODO in the code for this as well.